### ISSUE TITLE
Parameter scheduler

### ISSUE THREAD

**Issue by danieltudosiu** [Project-MONAI/MONAI#1552]
**Is your feature request related to a problem? Please describe.**
There is no elegant way from MONAI directly to have lambdas in losses be scheduled or network parameters being scheduled (for example the decay in VQ-VAEs).

**Describe the solution you'd like**
A parameter scheduler handler based around the closed forms available in PyTorch learning rate schedulers.

**Additional context**
I already have most of the code done. I just need to test it.

**Comment by Nic-Ma** [comment]
Hi @danieltudosiu ,

Thanks for your interest here, could you please help submit a PR for your idea?
@wyli , could you please help double confirm the request?

Thanks.

**Comment by danieltudosiu** [comment]
I did not forget about this feature request, atm I do not have the bandwidth to do a PR even though I have the core already implemented. I think in a month or so you can expect a PR from me. Otherwise, I am happy to put my working code here and you can use it.

**Comment by danieltudosiu** [comment]
@wyli Is there time for me to do a pull request for this?

Also, is this still of interest for MONAI?

**Comment by wyli** [comment]
yes, perhaps not for v0.5 but please send a pull request if you have time...

**Comment by danieltudosiu** [comment]
Ok. you will have it today.
